contracts/attribute/OwnerExecutable.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import { Ownable } from "./Ownable.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+/**
+ * @title OwnerExecutable
+ * @notice Allows the owner to execute calls to other contracts
+ */
+abstract contract OwnerExecutable is Ownable {
+    using Address for address;
```

Comment:
Couple notes on this:
- We try to avoid using `using` at the contract level, instead preferring to create custom types and assign logic through the `using global` modifier (as in `root`), or call libraries directly.
- Since we're only doing one call to `Address` I think it's a bit cleaner to make a direct call to the library instead of polluting the type: `Address.functionCallWithValue(target, data, msg.value);`